db/migrate/20160613221916_add_password_digest_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPasswordDigestToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :password_digest, :string
```

Comment:
Yes, Eli showed me how to pinpoint the exact db:migration to rollback in order to re run the command. Thanks.